reactor-core/src/main/java/reactor/core/publisher/UnicastManySinkNoBackpressure.java

```diff
@@ -67,13 +70,34 @@
 	public void subscribe(CoreSubscriber<? super T> actual) {
 		Objects.requireNonNull(actual, "subscribe");
 
-		if (!STATE.compareAndSet(this, State.INITIAL, State.SUBSCRIBED)) {
-			Operators.reportThrowInSubscribe(actual, new IllegalStateException("Unicast Sinks.Many allows only a single Subscriber"));
-			return;
+		State s;
```

Comment:
nit: Could use a similar style as in https://github.com/reactor/reactor-core/blob/56e5c0ddf8e9485b9117c75795db0afcd64eef6e/reactor-core/src/main/java/reactor/core/publisher/FluxConcatMapNoPrefetch.java#L226